### PR TITLE
Update to 4.2 but still compatible with 4.1

### PR DESCRIPTION
**Pull request by startupz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "license": "OSL-3.0",
     "require": {
         "payplug/payplug-php": "^3.1",
-        "oro/commerce": "4.1.*"
+        "oro/commerce": "4.1.* || 4.2.*"
```
Comment (dan-payplug) [inline composer.json]:
```suggestion
        "oro/commerce": "^4.1.*"
```

**Review by dan-payplug — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (dan-payplug) [reply composer.json]: C'est possible non ?

**Review by dan-payplug — APPROVED** [review]

**Comment by Dnd-Smash** [comment]
Hello,

Do you have an estimation for the release date for this update by any chance ?

Many thanks,

**Comment by dan-payplug** [comment]
Hello @Dnd-Smash,

We will submit our new version today so it will be available probably during next week depending of Orocommerce reviewing process.

Thank you :)

**Pull request merged**